tests/testAutopas/tests/utils/MemoryProfilerTest.cpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file MemoryProfilerTest.cpp.cc
+ * @author F. Gratl
+ * @date 23.03.21
+ */
+
+#include "MemoryProfilerTest.h"
+
+#include "autopas/utils/MemoryProfiler.h"
+
+/**
+ * Make sure the memory profiler finds any memory usage.
+ */
+TEST(MemoryProfilerTest, currentMemoryUsage) {
+  EXPECT_GT(autopas::memoryProfiler::currentMemoryUsage(), 0)
+      << "Memory profiler did not find any memory usage! Typically this test should report ~3300kb.";
```

Comment:
why should this report 3300kb?
It isn't explained anywhere.
```suggestion
      << "Memory profiler did not find any memory usage! As gtest allocates some static memory, there should be at least some memory used.";
```